pkg/bbgo/serialmarketdatastore.go

```diff
@@ -30,24 +42,98 @@ func (store *SerialMarketDataStore) Subscribe(interval types.Interval) {
 	store.Subscription = append(store.Subscription, interval)
 }
 
-func (store *SerialMarketDataStore) BindStream(stream types.Stream) {
-	stream.OnKLineClosed(store.handleKLineClosed)
+func (store *SerialMarketDataStore) BindStream(ctx context.Context, stream types.Stream) {
+	if store.UseAggTrade {
+		go store.tickerProcessor(ctx)
```

Comment:
note this is not compatible with backtest

Reply:
yes, right now should only be used in real trade.

Reply:
add checks and fallback behaviour